static/src/stylesheets/module/email/_header.scss

```diff
@@ -62,3 +62,15 @@
     }
 }
 
+// PLAIN MODS
```

Comment:
I'm still confused but I'm not 100% up on BEM terminology, maybe run it by someone else to confirm it makes sense!